tests/vmi_networking_test.go

```diff
@@ -729,13 +729,32 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 					return vmi, nil
 				}
 
+				configureIpv6 := func(vmi *v1.VirtualMachineInstance) error {
```

Comment:
But you can configure this though cloud-init which is used anyway. Or did you find a problem with this?

Reply:
It is a temporary method till the RA PR is merged. I couldn't find a way to invoke dhclient using cloudinit (without RA).